src/init_IntervalArithmetic.jl

```diff
@@ -0,0 +1,53 @@
+@reexport using IntervalArithmetic
```

Comment:
This is quite complex, i wish we can keep the latest and greatest version :)

Reply:
I only moved the code here (exactly because I thought it was quite complex).